khelo/firebase.json

```diff
@@ -0,0 +1 @@
+{"flutter":{"platforms":{"android":{"default":{"projectId":"khelo-dev-8040e","appId":"1:92999931839:android:1ac9d94bec982f25ac330a","fileOutput":"android/app/google-services.json"}},"ios":{"default":{"projectId":"khelo-dev-8040e","appId":"1:92999931839:ios:f166ae55796f68ceac330a","uploadDebugSymbols":false,"fileOutput":"ios/Runner/GoogleService-Info.plist"}},"macos":{"default":{"projectId":"khelo-dev-8040e","appId":"1:92999931839:ios:06a5606f88805af9ac330a","uploadDebugSymbols":false,"fileOutput":"macos/Runner/GoogleService-Info.plist"}},"dart":{"lib/firebase_options.dart":{"projectId":"khelo-dev-8040e","configurations":{"android":"1:92999931839:android:1ac9d94bec982f25ac330a","ios":"1:92999931839:ios:f166ae55796f68ceac330a","macos":"1:92999931839:ios:06a5606f88805af9ac330a","web":"1:92999931839:web:700f24d4723a6db3ac330a","windows":"1:92999931839:web:a03c76720fe6248cac330a"}}}}}}
```

Comment:
Is this file auto-generated? I think we should add it to .gitignore as our project is open source